src/main/java/blackJack/domain/Card/Cards.java

```diff
@@ -0,0 +1,72 @@
+package blackJack.domain.Card;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static blackJack.domain.Card.CardFactory.INIT_CARD_SIZE;
+
+public class Cards {
+    private static final int BUST_LINE = 21;
+    private static final int EXTRA_SCORE = 10;
+
+    private final Set<Card> cards;
+
+    public Cards(List<Card> deck) {
+        this(new HashSet<>(deck));
+    }
+
+    private Cards(Set<Card> deck) {
+        this.cards = new HashSet<>(deck);
+    }
+
+    public Set<Card> getCards() {
+        return new HashSet<>(cards);
+    }
+
+    public boolean add(Card card) {
+        return cards.add(card);
+    }
+
+    public int getScore() {
+        int score = cards.stream().mapToInt(card -> card.getNumber().getValue()).sum();
```

Comment:
가독성을 위해 `.` 기준으로 개행해보면 어떨까요?
```suggestion
        int score = cards.stream()
                .mapToInt(card -> card.getNumber().getValue())
                .sum();
```